tests/export/test_html.py

```diff
@@ -1678,3 +1817,41 @@ def test_with_line_break(self):
 
         expected_html = '<p><a href="http://google.com">li<br />nk</a>.</p>'
         self.assert_document_generates_html(document, expected_html)
+
+
+class NumberingTestCase(DocumentGeneratorTestCase):
+    def test_lowerLetter_numbering_format_is_handled(self):
+        num_id = '2'
+        numbering_xml = '''
+            <num numId="{num_id}">
```

Comment:
No, it's not. I added support for handling numbering definitions to the DocumentGeneratorTestCase. It didn't make sense to do that without having at least one test case in the pydocx repo that used it, so I added one.